functions.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+/**
+ * Draws elements sequentially in formatted HTML.
+ *
+ * @param array $paintings --> an array of elements from our MySQL DB.
```

Comment:
doesnt always have to be from MySql Db - you might change the db source in the future. remove any mention of db